std_msgs/Empty emtpy
---
bool success